expertsystem/schemas/xml/particle_list.xml.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://github.com/ComPWA/expertsystem/blob/master/schemas/xml/particle_list.xml.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Validation schema for a YAML/JSON file directly converted from particle_list.xml",
+  "description": "Useful in combination with a GUI that can use this schema to validate wile reworking it",
+  "type": "object",
+  "required": ["ParticleList"],
+  "additionalProperties": false,
+  "properties": {
+    "ParticleList": {
+      "type": "object",
```

Comment:
Well, this is where things become a mess. I don't really use this schema, because a JSON schema cannot validate an XML schema (at least not in an IDE) -- I only use JSON schemas to validate the `dict` structures that come out of `xmltodict`. But I did use this `particle_list.xml.json` schema to validate the current `particle.DATABASE` instance _dumped to a YAML file_ in the IDE. This `particle_list.xml.json` does therefore not exactly reflect the structure of `particle_list.xml`.

Anyway... maybe the conclusion is, lets just remove this JSON schema? :grimacing:

Reply:
No, the array is one level lower, under `Particle`:
https://github.com/ComPWA/expertsystem/blob/a104dd5148b599f41dfdddf4935e2c5adc02baa6/expertsystem/particle_list.xml#L3-L6

Reply:
But i thought that loading the xml file has the same structure as the `DATABASE` instance

Reply:
No, `DATABASE` looks like a `Dict[str, dict]`, where `str`, is the particle name, and `dict` has the nested structure as defined by `xml/particle.json`.

Reply:
Ah ok